### PR TITLE
Remove wantErrFormatted and regenerate devkeys

### PR DESCRIPTION
**Pull request by deeglaze**

### CODE REVIEW

**Review by AdamCDunlap — APPROVED** [review]

**Comment by deeglaze** [comment]
Tests failing since the dev signing key expired September 1st. Regenerated.

File: sign/gcsca/gcsca.go [hunk 1]
```diff
@@ -161,31 +154,48 @@ func (ca *CertificateAuthority) Finalize(ctx context.Context, m styp.Certificate
...
+			case *overwritten:
+				names = append(names, n.name)
+			}
+		}
 	}
 
 	// update the root cert if needed
 	if mut.rootCert != nil {
-		wrote, err := ca.writeIfAllowed(ctx, ca.RootPath, certPemBytes(mut.rootCert))
+		fmt.Println("Root cert path", ca.RootPath)
```
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:183]: is this debug string useful?

File: sign/gcsca/gcsca.go [hunk 2]
```diff
@@ -197,13 +207,13 @@ func (ca *CertificateAuthority) writeIfAllowed(ctx context.Context, path string,
 	}
```
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:207]: Add comment on this function that at least documents what the bool means.

File: sign/gcsca/gcsca.go [hunk 3]
```diff
@@ -318,36 +328,52 @@ func certPemBytes(cert *x509.Certificate) []byte {
...
+
+type newobj struct {
+	name string
+}
+
 // upload uploads the given x.509 certificate for the named key to the GCS private bucket,
-// and updates the given manifest to reflect the created object's entry. Returns whether any writes
-// happened and any errors.
-func (ca *CertificateAuthority) upload(ctx context.Context, manifest *cpb.GCECertificateManifest, keyVersionName string, cert *x509.Certificate) (bool, error) {
+// and updates the given manifest to reflect the created object's entry. Returns whether any manifest
```
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:340]: Comment is out of date -- no longer "returns whether any manifest changes happened"

File: sign/gcsca/gcsca.go [hunk 4]
```diff
@@ -318,36 +328,52 @@ func certPemBytes(cert *x509.Certificate) []byte {
...
 			ObjectPath:     name,
 		})
 		manifest.Entries = entries
 	}
-	return wrote, nil
+	if exists {
+		fmt.Println("OVERWRITE", name)
+		return &overwritten{name: name}, nil
+	}
+	fmt.Println("NEWOBJ", name)
```
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:375]: debug print?

File: sign/gcsca/gcsca.go [hunk 5]
```diff
@@ -318,36 +328,52 @@ func certPemBytes(cert *x509.Certificate) []byte {
...
+	if entry == nil {
 		entries := append(manifest.Entries, &cpb.GCECertificateManifest_Entry{
 			KeyVersionName: keyVersionName,
 			ObjectPath:     name,
 		})
 		manifest.Entries = entries
 	}
-	return wrote, nil
+	if exists {
+		fmt.Println("OVERWRITE", name)
```
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:372]: debug print?

File: sign/gcsca/gcsca.go
Comment (AdamCDunlap) [inline sign/gcsca/gcsca.go:1]: I don't see how the changes in this file reflect the commit message. Can you make sure the commit messages accurately reflect all the changes you're making and also why you're making them?

**Review by AdamCDunlap — CHANGES_REQUESTED** [review]

File: sign/gcsca/gcsca.go
Comment (deeglaze) [reply sign/gcsca/gcsca.go:1]: Yes, I unfortunately didn't amend the commit messages well enough, though I did try to keep different concepts in different commits. I'll clean up the code from your other comments in a follow-up PR.

**Pull request merged**